<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="TeensyConnectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Teensy</string>
    </property>
   </widget>
   <widget class="QLabel" name="TeensyStatusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>74</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Disconnected</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OuputWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>583</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutputWindowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PopulateSongListButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Populate Song List</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SelectedStimuliList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PresetsLabel">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Presets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFPresetButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>110</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bengalese Finch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZFPresetButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>110</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Zebra Finch</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>0</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentStimulusLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Stimulus:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BeginExperimentButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Experiment</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
